libr/asm/arch/arm/armass64.c

```diff
@@ -624,6 +661,41 @@ if (op->operands[1].reg_type == ARM_REG64) {
 	return seq_data;
 }
 
+static ut32 and(ArmOp *op, bool is_bic) {
+	ut32 data = UT32_MAX;
+
+	// Reg types need to match
+	if (!(op->operands[0].reg_type == op->operands[1].reg_type)) {
```

Comment:
op->operands[0].reg_type is dereferenced 4 times. just hold it in a variable at the begining.

Reply:
for example: `ut32 op0_regtype = op->operands[0].reg_type;`